lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -214,8 +227,9 @@ class ViewConfiguration {
     GestureSettings? gestureSettings,
     List<DisplayFeature>? displayFeatures,
   }) {
+    assert(view == null || window == null);
```

Comment:
Personally, I'd use runtime check to make absolutely sure that no production code ends up with both being non-`null`. Asserts can be ignored. I assume the check will be removed against when `window` is removed.